docs/release-process/instructions.md

```diff
@@ -0,0 +1,85 @@
+# How to use the release tooling
+
+## Creating the tracking issue
+
+First, create an issue on the microsoft/go repository for release progress updates. When you pass the issue number to the release automation pipeline, that makes the pipeline post a comment on that issue to notify you when a step completes or fails.
+
+If dealing with multiple releases at the same time (for example, 1.17 and 1.18 security patches released on the same day), create one issue per version number. This makes it easier to keep track of each release's progress as an outside observer.
+
+> The "release-start" pipeline is planned to create these issues automatically for common servicing events. (See [README.md](README.md).)
+
+## Upstream release
+
+After an upstream release, or in anticipation of an upstream release, go to the [`microsoft-go-infra-release-build`](https://dev.azure.com/dnceng/internal/_build/results?buildId=1780270&view=results) pipeline. Press "Run new" and fill in the blanks. For example, a 1.42.3-1 patch release would look like this:
+
+> ![](images/run-upstream-release.png)
+
+> Even though AzDO doesn't show the text selection cursor, you can actually click and drag to select `go-release-config` and copy-paste it into the text box.
+>
+> It may seem strange that `go-release-config` isn't the default. But:
+>
+> * For testing, you can pass in a different variable group. Requiring this field to be filled in each time makes sure you intend to run a real release.
+> * This makes infrastructure safer: if the build gets triggered and the set of parameters isn't passed correctly by tooling, the build fails safe by refusing to start.
+
+Once the values are filled in, press Run.
+
+You can close the tab as soon as the build starts looking for an agent. If nothing goes *terribly* wrong, the build will notify you if it needs intervention by commenting on the release issue.
+
+## microsoft/go revision
+
+Sometimes, we have to release when upstream hasn't made any changes. For example, to fix an issue with the FIPS implementation, or Go being built incorrectly by our infrastructure.
+
+In this case, run release-build, and increment the revision number from what it was for the latest release. If 1.18.3-1 was the latest, use 1.18.3-2 as the version.
```

Comment:
This actually needs more intervention with the current pipeline. It doesn't update MICROSOFT_REVISION, so the build asset JSON will be wrong, resulting in incorrect URLs and go-images tag names.

* https://github.com/microsoft/go/issues/575

I'm going to work on this now, and we can hold off on merging the doc until then.

Similar goes for any boring branch release (pre 1.19):

* https://github.com/microsoft/go/issues/470